client/src/components/IncomeTracker/IncomeTracker.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { IncomeItem, BudgetNavbar } from "../../components";
+import { BsPlusCircle } from "react-icons/bs";
+import {
+  Container,
+  IncomeContainer,
+  TitleContainer,
+  Title,
+  Button,
+} from "./IncomeTracker.styles";
+
+interface IncomeData {
+  id: number;
+  title: string;
+  amount: number;
+}
+
+interface Props {
+  setDisplayAdder: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+// Static dummy DATA for example list
+const dummyIncomeData: Array<IncomeData> = [
+  {
+    id: 1,
+    title: "Weekly Check",
+    amount: 1200,
+  },
+  {
+    id: 2,
+    title: "Tax Return",
+    amount: 3500,
+  },
+  {
+    id: 3,
+    title: "Gifted",
+    amount: 600,
+  },
+];
+
+const IncomeTracker: React.FC<Props> = ({ setDisplayAdder }) => {
```

Comment:
You missed this props-interface. Maybe name it IncomeTrackerProps?

Reply:
Ah, I did miss one! Tricky tricky lol